gateway/handle_http_auth_test.go

```diff
@@ -76,6 +77,21 @@ func TestAuth(t *testing.T) {
 			body, err := io.ReadAll(w.Body)
 			require.NoError(t, err, "reading response body should succeed")
 			require.Equal(t, "Failed to read writeKey from header\n", string(body))
+			require.Equal(t,
+				float64(1),
+				statsStore.Get(
+					"gateway.write_key_requests",
+					map[string]string{
```

Comment:
[optional]: Also at other below places as well.
```suggestion
					stats.Tags{
```